### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by harsitagarwalla187**
#513 

In this Pull Request, I have added RDS logo, Sign in with Github Button and (Welcome, Events, Members and Status) hyperlinks in the Navbar.
I have also taken care of responsiveness of the Navbar.
Screenshot 2023-09-06 201405
Screenshot 2023-09-06 201418
[Watch the Video](https://www.loom.com/share/aed58727c5004fc1b52cf89d770c129f?sid=04f95eae-d0c1-4949-9428-21a958918657)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -24,7 +24,62 @@ body {
...
   background-color: var(--blue-color);
   padding: 10px;
-  height: 2.5rem;
+  display: flex;
+}
+
+img {
+  height: 100%;
+  width: 100%;
+}
```
Comment (pallabez) [inline style.css:33]: Modifying global css doesn't seem right. What do you think?

Comment (harsitagarwalla187) [reply style.css:33]: I think it will protect every images from overflow and i couldn't think of any problems.

File: style.css [hunk 2]
```diff
@@ -295,3 +350,54 @@ footer {
...
+    flex-direction: column;
+    background-color: var(--white-color);
+    width: 100%;
+    padding: 10px;
+  }
+  
+  .nav-links .links a {
+    color: var(--blue-color);
+  }
+
```
Comment (pallabez) [inline style.css:402]: When working with media query, it's better to have as little things inside media query as possible.

From the functionality I see, you would need to only use - 

display: none/block
&
justify-content/align-items: center

**Comment by pallabez** [comment]
Thanks for the PR. The site really looks good now.

File: style.css (hunk 2 above)
Comment (harsitagarwalla187) [reply style.css:402]: Ok. Shall I make the changes?

**Comment by Achintya-Chatterjee** [comment]
Please look at the tests. Tests are failing. Please look into that

File: style.css (hunk 2 above)
Comment (pallabez) [reply style.css:402]: Yes please

**Comment by harsitagarwalla187** [comment]
@pallabez @Achintya-Chatterjee Can you please check it now?

**Review by pallabez — APPROVED** [review]
LGTM

**Review by Achintya-Chatterjee — DISMISSED** [review]
LGTM

**Review by pallabez — APPROVED** [review]

**Pull request merged**